src/app/pages/storage/storage.page.scss

```diff
@@ -1,7 +1,43 @@
-ion-thumbnail {
-  --size: 25vw;
+
+.img-card {
+  margin: calc( var( --ion-padding, 9px ) / 3 );
```

Comment:
Codacy found an issue: [Unexpected whitespace after "(" in a single-line function (function-parentheses-space-inside)](https://app.codacy.com/gh/numbersprotocol/capture-lite/pullRequest?prid=6209019)